### PR TITLE
2387: Remove jest errors

### PR DESCRIPTION
**Pull request by sarahsporck**
2387: add mock for react-native-svg

### Short description

Fix errors in tests.

### Proposed changes

- Add mock for for SVG Uri
- Add missing react-i18next mock

### Resolved issues

Fixes: #2387

### CODE REVIEW

File: native/src/components/SimpleImage.tsx [hunk 1]
```diff
@@ -30,12 +30,14 @@ const SimpleImage = ({
   resizeMode = 'contain',
   specifyAspectRatio = false,
 }: SimpleImageProps): ReactElement => {
-  const [aspectRatio, setAspectRatio] = useState<number | undefined>(undefined)
-
-  useEffect(() => {
+  const aspectRatio = useMemo(() => {
```
Comment (sarahsporck) [inline native/src/components/SimpleImage.tsx:33]: `useMemo` is resolved before the first render while `useEffect` is only run after the first render. This leads to weird test behavior. Also useMemo seems to be the right choice here, as we just assign a value.

**Review by steffenkleinle — APPROVED** [review]
Looks good :+1: Tested on android

**Comment by steffenkleinle** [comment]
@sarahsporck do you think we could not log error messages from console.error in the console for the tests? Like sometimes we are testing what happens if an error is thrown and that clutters the logs. Not related to the issue though.

**Comment by sarahsporck** [comment]
> @sarahsporck do you think we could not log error messages from console.error in the console for the tests? Like sometimes we are testing what happens if an error is thrown and that clutters the logs. Not related to the issue though.

What a throwback to one of my first tickets at digitalfabrik -> https://github.com/digitalfabrik/integreat-webapp/pull/366
Not sure why we removed the workaround completely though. I also don't see a reason to not override `console.error` for tests. Feel free to create a ticket :)

**Pull request merged**